src/pages/user/UserRegister.jsx

```diff
@@ -52,7 +52,7 @@ const UserRegister = () => {
       email: { type: "string", format: "email" },
       password: { type: "string", minLength: 8 },
       address: { type: "string" },
-      pincode: { type: "number", pattern: "[0-9]+", minLength: 6 },
+      pincode: { type: "string", pattern: "[0-9]+", minLength: 6 , maxLength: 6},
```

Comment:
I think the type should have been `number` as for most of the cases it is in number format.

Reply:
If I change it to a number, two errors will be shown.

![image](https://github.com/tamalCodes/Milan/assets/100359818/5ac86856-5636-4cc7-bcec-7e971131ab1a)

Pincode is not a valid value because of the datatype of pincode as passing it as "Number"  but to measure its length we need to send it as String!

If showing two errors works for you then I will make the changes. Kindly let me know.

Reply:
@Sar-thak-3 , there should not be 2 errors obviously. 

At first You can put it in string and limit the input to 6. 
And if the value is within the range of 6 digits/char - convert it to a number of 6 digits and then push it to the DB.